src/test/java/com/baidu/hugegraph/test/util/ClientUtil.java

```diff
@@ -42,18 +42,18 @@ public ClientUtil(String url, String graph, String username,
         this.timeout = timeout;
         this.trustStoreFile = trustStoreFile;
         this.trustStorePassword = trustStorePassword;
-        this.client();
+        this.hugeClient();
     }
 
-    protected void client() {
+    protected void hugeClient() {
         this.hugeClient = HugeClient.builder(this.url, this.graph)
                                     .configUser(this.username, this.password)
                                     .configTimeout(this.timeout)
-                                    .configSSL(trustStoreFile, trustStorePassword)
+                                    .configSSL(this.trustStoreFile, this.trustStorePassword)
                                     .build();
     }
 
-    public HugeClient hugeClient() {
+    public HugeClient client() {
```

Comment:
prefer to delete ClientUtil class since there is no more features than HugeClient